tests/integration/servicemesh/test-templates/authorino/mesh-authz-ext-provider.patch.tmpl

```diff
@@ -0,0 +1,13 @@
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: {{ .ControlPlane.Name }}
+  namespace: {{ .ControlPlane.Namespace }}
+spec:
+  techPreview:
+    meshConfig:
+      extensionProviders:
+      - name: {{ .AppNamespace }}-odh-auth-provider
```

Comment:
I think it would be good to make it distinctive not only in source/location of the file used in test but also in the content, so we can see it was applied.

```suggestion
      - name: {{ .AppNamespace }}-odh-different-auth-provider
```